demo/kserve/create-minio.md

```diff
@@ -1,27 +1,44 @@
-# Create your own Minio image
+# Create your own MinIO image
 
-If you would to create your own minio image with a desired model, first fork https://github.com/kserve/modelmesh-minio-examples.
+You can create your own MinIO image with a model.
 
-1. Clone the repo (update `<YOUR-USERNAME>`) and navigate to the correct directory.
-~~~
-git clone https://github.com/<YOUR-USERNAME>/modelmesh-minio-examples.git
-cd modelmesh-minio-examples
-~~~
+**Prerequisite**
 
-2. Create your model directory (update `<yourmodeldirectory>`) 
-~~~
-mkdir <yourmodeldirectory> && cd <yourmodeldirectory>
-~~~
-and place your model there.
+* You created a fork of the https://github.com/kserve/modelmesh-minio-examples GitHub repository.
 
+* You created a model.
 
-3. Edit the Dockerfile (located in the main directory) to add lines for your directory following the pattern [here](https://github.com/kserve/modelmesh-minio-examples/blob/main/Dockerfile#L36) and [here](https://github.com/kserve/modelmesh-minio-examples/blob/main/Dockerfile#L59).
+* You have a GitHib account.
 
-4. Build the image:
-~~~
-podman build -t quay.io/`<YOUR_QUAY_USERNAME>`/modelmesh-minio-examples:`<YOUR_TAG>` .
-~~~
-5. Push the image to quay:
-~~~
-podman push quay.io/`<YOUR_QUAY_USERNAME>`/modelmesh-minio-examples:`<YOUR_TAG>`
-~~~
\ No newline at end of file
+* You have a Quay account.
+
+**Procedure**
+
+1. Clone the repo (replace `<YOUR-USERNAME>` with your GitHub username).
+   ~~~
+   git clone https://github.com/<YOUR-USERNAME>/modelmesh-minio-examples.git
+   ~~~
+
+2. Navigate to the `modelmesh-minio-examples` directory.
+   ~~~
+   cd modelmesh-minio-examples
+   ~~~
+
+3. Create a model directory (replace `<yourmodeldirectory>` with the name of a directory for your model).
+   ~~~
+   mkdir <yourmodeldirectory>
+   ~~~
+
+4. Copy your model into the `modelmesh-minio-examples/<yourmodeldirectory>` directory .
+
+5. Edit your local `main/Dockerfile` and add lines for your model directory by following the pattern [here](https://github.com/kserve/modelmesh-minio-examples/blob/main/Dockerfile#L36) and [here](https://github.com/kserve/modelmesh-minio-examples/blob/main/Dockerfile#L59).
+
```

Comment:
@heyselbi I did not understand this step - perhaps we need to provide an example?
If your model directory name is *yourmodel* I need to add these two lines: 

COPY --chown=1000:0 yourmodel    ${MODEL_DIR}/yourmodel/

COPY --chown=1000:0 yourmodel     ${FVT_DIR}/yourmodel

Reply:
Yess, I would also add `<...>`:
COPY --chown=1000:0 <yourmodel> ${MODEL_DIR}/<yourmodel>/

COPY --chown=1000:0 <yourmodel> ${FVT_DIR}/<yourmodel>